app/assets/scss/application.scss

```diff
@@ -81,18 +72,25 @@ $path: "/static/images/";
 @import "_table.scss";
 @import "_text.scss";
 
+// Misc styles
+// TODO: Move misc styling into their own appropriate files to import or the Digital Marketplace FE Toolkit
```

Comment:
We could just do this now rather than leave it as a TODO?

Reply:
I can do one or two of these now I think. However, at the moment I would only be moving the rest into a misc file (which feels bad) because the majority of these need a bit of refactoring/shouldn't really exist which would take a little bit of time (like thinking how all our typography is structured).

`padding-bottom-small` is not semantic and was some styling copied from the prototype which we shouldn't really be using. Not sure how best to replace this at the moment.
`return-to-top` I could move to the service-page.scss as I think it is just used at the bottom of service pages
`marketplace-paragraph` is probably another candidate from removal as we should be using typography for this.
`break-link` should potentially go into the toolkit as we write the HTML for it in the utils repo.
`frameworks-lots` is just used for the g cloud index page. Not sure if it's worth creating a single file just for this one at the moment.

So I will move the easy one but leave the rest. Does that sound reasonable or am I being lazy?

Reply:
I think framework lots is fine to go into a separate file (although it's actually meant to be an update to our general styling of bulleted lists so it _should_ go to the fe-toolkit eventually... as we are/were heading towards removing hanging bullets).

The others, yeah, I guess can stay. Looking into marketplace-paragraph/dmspeak usage in this app, there's clearly work to be done here...